wgpu-hal/src/dx12/device.rs

```diff
@@ -1260,10 +1260,10 @@ impl crate::Device<super::Api> for super::Device {
     }
     unsafe fn destroy_bind_group(&self, group: super::BindGroup) {
         if let Some(dual) = group.handle_views {
-            let _ = self.shared.heap_views.free_slice(dual);
+            self.shared.heap_views.free_slice(dual);
         }
         if let Some(dual) = group.handle_samplers {
-            let _ = self.shared.heap_samplers.free_slice(dual);
+            self.shared.heap_samplers.free_slice(dual);
```

Comment:
This isn't really related to this PR, is it?

Reply:
This is the outstanding lint that wasn't caught by the previous edition of the CI, so I had to fix it to get this CI to pass